tests/test_commit_command.rs

```diff
@@ -54,6 +54,25 @@ fn test_commit_with_editor() {
         std::fs::read_to_string(test_env.env_root().join("editor0")).unwrap(), @r###"
     initial
 
+    JJ: This part contains the following changes:
```

Comment:
We should probably special-case the empty diff and either skip this output or say something like this:

```
JJ: This commit contains the following changes:
JJ: <none>
```

Can be done later if you prefer.

Reply:
Yep, removed the diff summary if commit/describe content is empty. I think that's common in "new -> describe -> do something" workflow.